projects/module-2/flash-cards.md

```diff
@@ -0,0 +1,247 @@
+  - Look at the file being run when we want to start the game. Think about where you need to invoke your `start` function.
+
+## Iteration 4:
+
+Choose one or more of these extensions for this iteration.
+
+_Note: Solving some of these problems may require modifying pre-existing code that was given to you_
+
+***Use a project board to guide your workflow***
+Use a tool like GitHub projects or Trello to split up iteration items into small tasks.
+
+***Build out another study/flashcards dataset***
+Create a new dataset that is structured similarly to our prototypes dataset. This dataset should be reviewing another fundamental topic and should live in the same `data.js` file. Update your application so that once the first `round` is over (and thus, the `game` is over), a new `round` starts with the new dataset that you've created.
+
+***More practice***   
+Add functionality so that a score of less than 90% will require the user to go through the entire dataset again.
+
+***Review incorrect answers***  
+Add functionality so that the `round` (and thus, the `game`) does not end after you've gone through all of the cards. The player should then be prompted to try to guess again, reviewing only those cards that are stored as incorrect.
+
+***Timer***  
+At the end of the game, report how much time it took to complete the game (in minutes and seconds).
+
+***Report Card***  
+At the end of the game, create a report card that would detail any answers that the user got wrong as well as how many tries each question took.
+
+***Options for mixed datasets***  
+At the beginning of the game, give options for choosing to study different subjects (have a dataset for one subject and another subject), as well as the option for mixing all of the subjects together.
+
+***CYOE (Choose Your Own Extension)***  
+Collaborate with instructors to personalize an extension for this project
+
+## README
+
+Your README should include the following, in this order:
+
+- Abstract at the top (A sentence or two describing the project)
+- A GIF of the working application
+- Technologies used
+- Install/Setup instructions
+
+## Rubric
+
+For the rubric sections below, you will be scored as **Wow**, **Yes** or **Not Yet** depending on whether you have demonstrated competency in that area. Each section lists examples of what types of things we may be looking for as demonstrations of competency. Just as there are many ways to approach code, there are many many ways to demonstate competency.  There are just some examples. 
+
+### Functional Expectations
+* Wow: Application fulfills all requirements *as well as* an extension from iteration 4.
+* Yes: Application fulfills all requirements of iterations 1-3 without bugs.
+* Not Yet: Application crashes (game is not playable) or has missing functionality or bugs.
+
+### JavaScript & Style / Functional Programming
+- Code is divided into logical components each with a clean, single responsibility
```

Comment:
In m1, we've added something like "On track looks like..." before the bullets for clarity in each rubric section

Reply:
Fixed!